code/modules/language/nekomimetic.dm

```diff
@@ -0,0 +1,14 @@
+/datum/language/nekomimetic
+	name = "Nekomimetic"
+	desc = "To the casual observer, this langauge is an incomprehensible mess of broken Japanese. To the felinids, it's somehow comprehensible."
+	key = "f"
+	space_chance = 70
+	syllables = list(
+		"neko", "nyan", "mimi", "moe", "mofu", "fuwa", "kyaa", "kawaii", "poka", "munya",
+		"puni", "munyu", "ufufu", "uhuhu", "icha", "doki", "kyun", "kusu", "nya", "nyaa",
+		"desu", "kis", "ama", "chuu", "baka", "hewo", "boop", "gato", "kit", "sune", "yori",
+		"sou", "baka", "chan", "san", "kun", "mahou", "yatta", "suki", "usagi", "domo", "ori",
+		"uwa", "zaazaa", "shiku", "puru", "ira", "heto", "etto"
+	)
```

Comment:
I am fine with felinid saying such weird syllables but the current pool shows too felinid-meme at high possibility. Adding more japanese-like syllables to lower awkward combination.
For example, having more syllables like `"itai", "omae", "akaiku"` bla bla would be more acceptive, and I hope that'll be less awkward. Such things shouldn't happen that often. It should come unexpectedly then strike you.